From 4c4741e0ad0206f7457f7ea0fbe1be981d1933c2 Mon Sep 17 00:00:00 2001
From: Christer <christer.bjor@gmail.com>
Date: Tue, 10 Mar 2015 14:36:35 +0100
Subject: [PATCH] hlte-common: remove bogus MMS overlay.

* This is already taken care of in samsung_qcom-common and is cuasing
data drops with some carriers after receiving an MMS as well as not
sending messages over 160 characters on some carriers.

Change-Id: I9f26bc33ed99515f15c881f697917d8c0c3c3cb6
---
 overlay/packages/apps/Mms/res/xml/mms_config.xml | 49 ------------------------
 1 file changed, 49 deletions(-)
 delete mode 100644 overlay/packages/apps/Mms/res/xml/mms_config.xml

diff --git a/overlay/packages/apps/Mms/res/xml/mms_config.xml b/overlay/packages/apps/Mms/res/xml/mms_config.xml
deleted file mode 100644
index 3d7cf29..0000000
--- a/overlay/packages/apps/Mms/res/xml/mms_config.xml
+++ /dev/null
@@ -1,49 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright (C) 2009 The Android Open Source Project
-
-     Licensed under the Apache License, Version 2.0 (the "License");
-     you may not use this file except in compliance with the License.
-     You may obtain a copy of the License at
-
-          http://www.apache.org/licenses/LICENSE-2.0
-
-     Unless required by applicable law or agreed to in writing, software
-     distributed under the License is distributed on an "AS IS" BASIS,
-     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-     See the License for the specific language governing permissions and
-     limitations under the License.
--->
-
-<!-- Version History
-        version 1 - initial version.
-        version 2 - added recipientLimit.
-        version 3 - added min/max recycler values.
--->
-
-<mms_config version="3">
-
-    <!-- Maximum message size in bytes for a MMS message -->
-    <int name="maxMessageSize">1045876</int>
-
-    <!-- Maximum height for an attached image -->
-    <int name="maxImageHeight">2592</int>
-
-    <!-- Maximum width for an attached image -->
-    <int name="maxImageWidth">2592</int>
-
-    <!-- If true, The text message over 160 characters will be sent in multi part.
-         If false, The text message over 160 characters will be sent
-         via multi media message. -->
-    <bool name="enableMultipartSMS">true</bool>
-
-    <!-- If true, the text message will be split every 160 characters.
-         If false, the text will never be split before being sent. -->
-    <bool name="enableSplitSMS">false</bool>
-
-    <!-- User-Agent parameter used in MMS http request -->
-    <string name="userAgent">SAMSUNG-GT-I9300-Mms</string>
-
-    <!-- UAProf URL -->
-    <string name="uaProfUrl">http://wap.samsungmobile.com/uaprof/GT-I9300.xml</string>
-
-</mms_config>
